cli/ping_test.go

```diff
@@ -67,7 +66,6 @@ func TestPing(t *testing.T) {
 
 		pty.ExpectMatch("pong from " + workspace.Name)
 		pty.ExpectMatch("✔ received remote agent data from Coder networking coordinator")
-		cancel()
 		<-cmdDone
```

Comment:
Might be missing something but I'm not sure why this context was being manually cancelled after being deferred? It was causing our post-ping work to get cancelled early.

Reply:
It was probably to start tearing down the command after it got the output it expected.  Might be better to expect the post-ping output rather than remove the `cancel()`